src/pages/resultsView/ResultsViewPageStore.ts

```diff
@@ -873,14 +873,16 @@ export class ResultsViewPageStore {
 
     readonly filteredAlterations = remoteData<ExtendedAlteration[]>({
         await:()=>[
-            this.unfilteredAlterations,
+            this.putativeDriverAnnotatedMutations,
+            this.annotatedMolecularData,
             this.selectedMolecularProfiles,
             this.defaultOQLQuery
         ],
         invoke:()=>{
             if (this.rvQuery.oqlQuery.trim() != "") {
+                const data = [...(this.putativeDriverAnnotatedMutations.result!), ...(this.annotatedMolecularData.result!)];
```

Comment:
just curious why these are in parens

Reply:
hmm i guess no reason, spread operator wouldnt apply to `this` right?